distributed/benchmarks/client.py

```diff
@@ -56,6 +56,7 @@ def setup(self,resource,steal_interval):
         self.client = client
 
     def teardown(self,resource,steal_interval):
+        self.cluster.close()
```

Comment:
Looks like we're not keeping the cluster as a class attribute. We could do this, or using `self.client.shutdown()` instead should be equivalent. 

```suggestion
        self.client.shutdown()
```